tests/components/homekit/test_homekit.py

```diff
@@ -933,6 +1018,7 @@ async def test_homekit_uses_system_zeroconf(hass, hk_driver, mock_zeroconf):
         data={CONF_NAME: BRIDGE_NAME, CONF_PORT: DEFAULT_PORT},
         options={},
     )
+    assert await async_setup_component(hass, "zeroconf", {"zeroconf": {}})
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)